### DISCUSSION TITLE
Issues with Suspended Account: No Email Notifications and Support Contact Not Working

### DISCUSSION THREAD

**Discussion by Wahid7852** [community/community#145004] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello everyone at the support team,

I’m currently facing a situation where my friend's (@techy4shri) GitHub account has been suspended, and I’ve encountered two major 

Despite my friends account being suspended, they have not received any email notifications regarding the suspension. I expected a notification to clarify the reasons behind the suspension and steps to resolve the issue, but so far, there has been no communication sent to my registered email address. Is there a specific reason why suspended account owners might not be receiving such emails? Shouldn't this be a standard practice to ensure 

They have also tried reaching out to GitHub Support regarding the suspension, but are faced with a 403 error (image attached)
Has anyone else experienced difficulties in contacting GitHub support when dealing with a suspended account? If so, are there alternative ways to get help with account issues?

I would greatly appreciate any insights or suggestions from the community on why this might be happening and how I can move forward to address these concerns.

![shri1](https://github.com/user-attachments/assets/d06bb8c1-e7f3-4edc-b8cd-e60975f09ffe)
![shri2](https://github.com/user-attachments/assets/9346014a-376a-4614-8633-f87b5de21378)

**Comment by mecodeatlas** [comment]
Hi @Wahid7852 

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!

**Comment by techydevi** [comment]
@mecodeatlas can you please tell why an account would be suspended with no email or any other communication provided with the support page for reinstatement being inaccessible?
